api/client/pause.go

```diff
@@ -18,7 +18,7 @@ func (cli *DockerCli) CmdPause(args ...string) error {
 	for _, name := range cmd.Args() {
 		if _, _, err := readBody(cli.call("POST", fmt.Sprintf("/containers/%s/pause", name), nil, nil)); err != nil {
 			fmt.Fprintf(cli.err, "%s\n", err)
-			encounteredError = fmt.Errorf("Error: failed to pause container named %s", name)
+			encounteredError = fmt.Errorf("Error: failed to pause one or more containers")
```

Comment:
let's just add the names of all the containers that failed. More info can be found on the daemon.